CHANGELOG.md

```diff
@@ -28,17 +43,19 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 - Install NVIDIA Fabric manager to enable NVIDIA NVSwitch on supported platforms.
 
 **CHANGES**
-- Upgrade EFA installer to version 1.10.1
-  - EFA configuration: ``efa-config-1.5`` (from efa-config-1.4)
-  - EFA profile: ``efa-profile-1.1`` (from efa-profile-1.0.0)
-  - EFA kernel module: ``efa-1.10.2`` (from efa-1.6.0)
-  - RDMA core: ``rdma-core-31.amzn0`` (from rdma-core-28.amzn0)
-  - Libfabric: ``libfabric-1.11.1amzn1.1`` (from libfabric-1.10.1amzn1.1)
-  - Open MPI: ``openmpi40-aws-4.0.5`` (from openmpi40-aws-4.0.3)
-  - Unifies installer runtime options across x86 and aarch64
-  - Introduces ``-g/--enable-gdr`` switch to install packages with GPUDirect RDMA support
-  - Updates to OMPI collectives decision file packaging, migrated from efa-config to efa-profile
-  - Introduces CentOS 8 support
+- Upgrade EFA installer to version 1.11.0
```

Comment:
Why are we changing the 2.10.0 changelog?

Reply:
Copy/paste error :) Thanks